src/main/java/org/commcare/formplayer/application/WebAppContext.java

```diff
@@ -247,6 +248,13 @@ public FormplayerSentry raven() {
         return raven;
     }
 
+    @Bean(destroyMethod="cleanup")
```

Comment:
What does this do?

Reply:
Ah definitely don't need this. Didn't refine my copy/paste for the `FormplayerSentry` object, but looks like it just specifies to call that method (`cleanup`) when destroying the object. The Datadog Client file doesn't have any special cleanup, so I don't think the `FormplayerDatadog` class needs any either.